mods/persistence/modules/mob/new_player.dm

```diff
@@ -131,75 +133,9 @@
 		qdel(src)
 		return
 
-	switch(alert("Are you sure you want to join the game with the character you've created? This cannot be undone!", "Character Confirmation", "Yes", "No"))
```

Comment:
Why was this check removed?

Reply:
Because its not true anymore. The player can still abort until they use the launch pod by disconnecting. I made it so if a player disconnects in the chargen area his character is deleted to prevent having to deal with them. Also prevents buildup of mobs in there.

Reply:
I think that disconnecting in the chargen area is pretty unintuitive. Drop the "This cannot be undone!" if you'd like but the confirmation is worth keeping, since on other servers pressing this button normally goes to select a role rather than dropping you right in the game.

Reply:
Its probably better to put that in the text chat after they spawn in chargen along with an explanation of what's going on. 
But I'm strongly against popping modal alerts dialogs for no reasons. They're annoying and clunky as hell. They've also been long established to only happen when something goes wrong on the start menu in SS13. Not when you're joining the game and everything is fine.

Reply:
With the alert, the expectation that something went wrong can be instantly resolved by just reading the alert. On the other hand, clicking the join game button expecting a "Select your role..." etc. prompt will just dump you in blind. For reference, this prompt was specifically added at the request of a player during the first open testing round.

Reply:
Its pretty common to reconnect to get back to the menu. That's just how the game is made right now.
Also, I'm not really into putting in a feature that breaks the flow of things for everyone just because a single person asked for it, and didn't provide explanations. Plus in the older builds you just couldn't go back, but now you can.